launcher/FileSystem.cpp

```diff
@@ -872,16 +831,35 @@ QString RemoveInvalidPathChars(QString path, QChar replaceWith)
     invalidChars = BAD_WIN_CHARS;
 #endif
 
-    QString fileSystemType = getFileSystemType(QFileInfo(path).absolutePath());
-
-    if (fileSystemType == "FAT32") {
-        invalidChars += BAD_FAT32_CHARS;
-    } else if (fileSystemType == "NTFS") {
-        invalidChars += BAD_NTFS_CHARS;
-    } else if (fileSystemType == "HFS+" || fileSystemType == "APFS") {
-        invalidChars += BAD_HFS_CHARS;
-    } else if (fileSystemType == "exFAT") {
-        invalidChars += BAD_EXFAT_CHARS;
+    switch (statFS(path).fsType) {
```

Comment:
Oh good, I was wondering why you were trying to reimpliment that function :P I only did hours of research to implement it :P